### PR TITLE
Updates Prettier to 1.5.0

### PR DESCRIPTION
**Pull request by orta**
https://github.com/prettier/prettier/releases/tag/1.5.0

* Adds JS in CSS
* Adds GraphQL support (but not for us, only on Relay Modern - ah well)

I experimented with adding style lint + [stylelint-processor-styled-components](https://github.com/styled-components/stylelint-processor-styled-components/) but there is no support for it in VS Code, so I've sent off a PR adding it https://github.com/shinnn/vscode-stylelint/pull/82

Once that's shipped then stylelint can be added /cc @l2succes 

Skip New Tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,32 +36,6 @@
...
-
-#### Consignments
-
-- [dev] Adds component for attaching buttons to the keyboard  - orta
-- [dev] Adds component for artist search  - orta
-- [dev] Adds component for consignments todo  - orta
-- [dev] Adds a root component for the Consignments flow  - orta
-- [dev] Adds storybooks for consignments  - orta
-- [dev] Adds some form elements for consignments - orta
-- Added the Welcome/Overview screen for consignments - orta
```
Comment (orta) [inline CHANGELOG.md]: This looks like someone's bad merge

File: .vscode/settings.json [hunk 2]
```diff
@@ -62,5 +62,9 @@
         "iPhone",
         "ListView",
         "ARTemporaryAPIModule"
+    ],
+    "stylelint.additionalDocumentSelectors": [
+        "typescript",
+        "typescriptreact"
```
Comment (orta) [inline .vscode/settings.json]: see https://github.com/shinnn/vscode-stylelint/pull/82 to understand

File: package.json [hunk 3]
```diff
@@ -111,7 +111,8 @@
     "jest-react-native": "18.0.0",
     "jest-snapshots-svg": "^0.0.14",
     "lint-staged": "^3.4.1",
-    "prettier": "^1.4.2",
+    "prettier": "^1.5.0",
+    "react-dom": "16.0.0-alpha.12",
```
Comment (orta) [inline package.json]: this was causing a tonne of warnings on the CLI when I ran storybooks, so I've added it in and updated storybooks too.

File: src/lib/Containers/__tests__/WorksForYou-tests.tsx [hunk 4]
```diff
@@ -86,21 +86,19 @@ interface NotificationsResponse {
...
+            artworks: [{ title: string }]
+            image: {
+              resized: {
+                url: string
               }
             }
           }
-        >
+        }>
       }
```
Comment (orta) [inline src/lib/Containers/__tests__/WorksForYou-tests.tsx]: These are better 👍

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  Missing Test Files:

- `src/lib/Components/Consignments/Components/__tests__/NavButton-tests.tsx`
- `src/lib/Components/Home/ArtistRails/__tests__/ArtistRail-tests.tsx`
- `src/lib/Components/Home/ArtworkRails/__tests__/ArtworkRailHeader-tests.tsx`
- `src/lib/Components/Home/__tests__/HeroUnits-tests.tsx`
- `src/lib/Components/Inbox/Conversations/Previews/__tests__/ArtworkPreview-tests.tsx`
- `src/lib/Components/RelatedArtists/__tests__/RelatedArtist-tests.tsx`

If these files are supposed to not exist, please update your PR body to include "Skip New Tests".
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  New dependencies added: react-dom.
  </td>
    </tr>
  </tbody>
</table>



<h2><a href="https://facebook.github.io/react/">react-dom</a></h2>
<p>Author: Unknown</p>
<p>Description: React package for working with the DOM.</p>
<p>Homepage: <a href="https://facebook.github.io/react/">https://facebook.github.io/react/</a></p>

<table>
  <thead><tr><th></th><th width="100%"></th></tr></thead>
  <tr><td>Created</td><td>about 3 years ago</td></tr><tr><td>Last Updated</td><td>6 days ago</td></tr><tr><td>License</td><td>BSD-3-Clause</td></tr><tr><td>Maintainers</td><td>2</td></tr><tr><td>Releases</td><td>73</td></tr><tr><td>Direct Dependencies</td><td><a href='http: //npmjs.com/package/fbjs'>fbjs</a>, <a href='http: //npmjs.com/package/loose-envify'>loose-envify</a>, <a href='http: //npmjs.com/package/object-assign'>object-assign</a> and <a href='http: //npmjs.com/package/prop-types'>prop-types</a></td></tr><tr><td>Keywords</td><td>react</td></tr>
</table>

<details>
<summary><code>README</code></summary>
# `react-dom`

This package serves as the entry point of the DOM-related rendering paths. It is intended to be paired with the isomorphic React, which will be shipped as `react` to npm.

## Installation

```sh
npm install react react-dom
```

## Usage

### In the browser

```js
var React = require('react');
var ReactDOM = require('react-dom');

class MyComponent extends React.Component {
  render() {
    return <div>Hello World</div>;
  }
}

ReactDOM.render(<MyComponent />, node);
```

### On the server

```js
var React = require('react');
var ReactDOMServer = require('react-dom/server');

class MyComponent extends React.Component {
  render() {
    return <div>Hello World</div>;
  }
}

ReactDOMServer.renderToString(<MyComponent />);
```

## API

### `react-dom`

- `findDOMNode`
- `render`
- `unmountComponentAtNode`

### `react-dom/server`

- `renderToString`
- `renderToStaticMarkup`

</details>




  <details>
    <summary><code>yarn why react-dom</code> output</summary>
    <p><code><ul><li>Has been hoisted to "react-dom"</li><li>This module exists because it's specified in "devDependencies".</li><li>Disk size without dependencies: "2.53MB"</li><li>Disk size with unique dependencies: "8.04MB"</li><li>Disk size with transitive dependencies: "16.63MB"</li><li>Number of shared dependencies: 13
    </li></ul></code></p>
  </details>
  
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: CHANGELOG.md (hunk 1 above)
Comment (mzikherman) [inline CHANGELOG.md]: twice the entries in the changelog, twice the visible amount of work, twice the $$ ??

File: src/lib/Components/Inbox/Bids/ActiveBid.tsx [hunk 5]
```diff
@@ -8,14 +8,13 @@ import { BodyText, MetadataText } from "../Typography"
...
-  margin: 17px 20px 0px;
-`
+const Container = styled.View`margin: 17px 20px 0px;`
 
 const Content = styled.View`
   flex: 1;
   flex-direction: row;
-  align-items: center;
+
+  \: center;
```
Comment (orta) [inline src/lib/Components/Inbox/Bids/ActiveBid.tsx]: looks like this is _hopefully_ a typo

**Comment by mzikherman** [comment]
Looks like some maybe real test failures?

**Comment by orta** [comment]
Fixed the test failures - thanks!

File: src/lib/Containers/__tests__/Conversation-tests.tsx [hunk 6]
```diff
@@ -26,7 +26,7 @@ const props = {
             raw_text: "Adoro! Por favor envie-me mais informações",
             from_email_address: "anita@garibaldi.br",
             attachments: [],
-            created_at: "2017-06-26T14:14:35.538Z",
+            last_message_at: moment().subtract(2, "days").toISOString(),
```
Comment (orta) [inline src/lib/Containers/__tests__/Conversation-tests.tsx]: This was causing the test to change each day

Comment (mzikherman) [inline src/lib/Containers/__tests__/Conversation-tests.tsx]: Ah shoot I think I added that the other day (and undid the correct relative timestamp). Thank ye

Comment (mzikherman) [inline src/lib/Containers/__tests__/Conversation-tests.tsx]: Did you mean to rewrite `created_at`?

Comment (orta) [inline src/lib/Containers/__tests__/Conversation-tests.tsx]: Ah - thanks!

Comment (orta) [inline src/lib/Containers/__tests__/Conversation-tests.tsx]: Shipped

**Comment by orta** [comment]
@mzikherman this is green

**Pull request merged**